jupyterhub_moss/templates/option_form.html

```diff
@@ -179,8 +179,22 @@ <h4 style="text-align: center">List of available resources:</h4>
       min="1"
       value="1"
     />
+    <label for="mem">
+      Memory <span class="label-extra-info">per node (--mem, max: <span id="max_memory_span"></span>)</span>:
+    </label>
+    <div class="form-field-div" title="Memory in GigaBytes">
+      <input type="hidden" id="mem" name="mem" value=""/>
```

Comment:
Using a hidden input to keep track of the ensemble "memory+unit" feels a bit convoluted but I can get around it.

Do you think having the unit in another input would help ?

Reply:
I wanted to have the same for the `mem` field in the POST request as the slurm `--mem` argument (simpler for the spawn through URL).
I hesitated with passing gigabytes and appending the `G` in the backend.

If you have a better option than this I'd take it, I agree it is convoluted.
It's maybe time to generate the POST request instead of sending the form inputs? But I'd do that in another PR.

> Do you think having the unit in another input would help ?

It sounds to me that a choice of unit is useless from a user point of view, I doubt one wants to use something else than Gigabytes.